guides/common/modules/proc_creating-a-content-view-filter-by-using-cli.adoc

```diff
@@ -1,34 +1,12 @@
 :_mod-docs-content-type: PROCEDURE
 
-[id="Creating_a_Content_Filter_for_Yum_Content_{context}"]
-= Creating a content filter for Yum content
+[id="creating-a-content-view-filter-by-using-cli"]
+= Creating a content view filter by using CLI
 
-You can filter content views containing Yum content to include or exclude specific packages, package groups, errata, or module streams.
+You can use Hammer CLI to create content view filters to include or exclude specific packages, package groups, or errata.
 Filters are based on a combination of the _name_, _version_, and _architecture_.
```

Comment:
Perhaps remove this line since it only applies to package filters?

Reply:
Dropped.